### PR TITLE
tidbServer : add `SetDBInfoMeta` for write table/database meta to storage directly.

### PR DESCRIPTION
**Pull request by AilinKid**
### What changed.
1 : Add `SetDBInfoMeta` for write table/database meta to storage directly.
2 : Export TiDBServer's storage by `GetStorage()`

### CODE REVIEW

File: tidb.go [hunk 1]
```diff
@@ -292,4 +277,74 @@ func DestoryTiDBServer(t *TiDBServer) {
...
+		 */
+		for i:=0; i<=105; i++ {
+			_, err := t.GenSchemaVersion()
+			if err != nil {
+				return errors.Trace(err)
+			}
+		}
+		return nil
+	})
+	return err
```
Comment (XuHuaiyu) [inline tidb.go]: We can put domain.Reload in this function

Comment (AilinKid) [reply tidb.go]: Addressed.

File: tidb.go [hunk 2]
```diff
@@ -292,4 +278,69 @@ func DestoryTiDBServer(t *TiDBServer) {
 	t.closeDomainAndStorage()
 	t.CloseGracefully()
 	tidbServer = nil
+}
+
+/*
+ * setDBInfoMeta / setTableInfoMeta is used to store the correct dbInfo and tableInfo into
```
Comment (XuHuaiyu) [inline tidb.go]: s/ `setDBInfoMeta / setTableInfoMeta` / `SetDBInfoMetaAndReload` ?

File: tidb_test.go [hunk 3]
```diff
@@ -65,4 +68,72 @@ func (t *testTiDBSuite) TestTiDBServer(c *C) {
...
+	var createTableString sql.NullString
+	for result.Next() {
+		err := result.Scan(&nameString, &createTableString)
+		c.Assert(err, IsNil)
+		break
+	}
+	c.Assert(nameString.String, Equals, "my_table")
+	//c.Assert(createTableString.String, Equals, "CREATE TABLE `my_table` (\n" +
+	//	"  `c01` int(11) CHARACTER SET  COLLATE  DEFAULT '0'\n" +
+	//    ") ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin")
```
Comment (WangXiangUSTC) [inline tidb_test.go]: we can delete these code?

File: .idea/.gitignore [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```
Comment (XuHuaiyu) [inline .idea/.gitignore]: We need `.gitignore` instead of `.idea/.gitignore`.

**Comment by WangXiangUSTC** [comment]
LGTM

**Pull request merged**